merkletree/str.go

```diff
@@ -56,6 +57,10 @@ func (str *SignedTreeRoot) Serialize() []byte {
 	return strBytes
 }
 
+// VerifyHashChain computes the hash of saved STR,
+// and compares it to the hash of previous STR included
```

Comment:
s/the hash of previous STR/the PreviousSTRHash (I think it's ok to just use the field name)